pkg/analysis_server/test/services/correction/fix_test.dart

```diff
@@ -6106,6 +6106,66 @@ final a = {};
 ''');
   }
 
+  test_tearOffLambda_function_test1() async {
```

Comment:
Fixed but having problems to create this test:

```dart
  test_tearOffLambda_lambda_asArgument() async {
    String src = '''
bool isPair(int a) => a % 2 == 0;
final finalList = <int>[];
finalList.where(/*LINT*/(number) =>
  isPair(number);
);
''';
    await findLint(src, LintNames.unnecessary_lambdas);

    await applyFix(DartFixKind.TEAR_OFF_LAMBDA);

    verifyResult('''
bool isPair(int a) => a % 2 == 0;
final finalList = <int>[];
finalList.where(isPair);
''');
  }
```

In this test the line

```dart
FunctionExpression ancestor =
  node.getAncestor((a) => a is FunctionExpression);
```

Is being resolved to null, do you know how to solve this?